app/[...missing].tsx

```diff
@@ -10,7 +10,7 @@ export default function NotFoundScreen() {
       <View className="flex-1 items-center justify-center p-5">
         <Text className="text-2xl font-bold">This screen doesn't exist.</Text>
 
-        <Link href="/index" className="mt-4 py-4">
+        <Link href="/" className="mt-15 py-15">
```

Comment:
This sort of styling will break on various screens, consider using auto margins instead